src_SSITH_P3/Makefile

```diff
@@ -102,16 +102,19 @@ Verilog_RTL:
 Verilog_RTL_sim:
 	mkdir -p $@
 
+../src_Core/BSV_Additional_Libs/BlueStuff/.git:
```

Comment:
This file still uses `$(REPO)` everywhere even though that's technically not needed, but makes it easier to follow

Reply:
This fix has been committed now, hopefully to your satisfaction.